package/network/services/hostapd/files/hostapd.sh

```diff
@@ -183,6 +183,8 @@ hostapd_common_add_bss_config() {
 	config_add_int maxassoc max_inactivity
 	config_add_boolean disassoc_low_ack isolate short_preamble
 
+	config_add_bolean skip_inactivity_poll
```

Comment:
`bolean` is a typo. and instead of adding this line and the following newline, please add `skip_inactivity_poll` after `short_preamble` above.

Reply:
Thanks for the spotting this one! Addressed both concerns raised in your comments above, and squashed the comments again.